extensions/deploy_replace_undeploy/standard/sections/clause_6_deploy_replace_undeploy.adoc

```diff
@@ -257,3 +257,28 @@ See <<deploy-replace-undeploy-http_status_codes,HTTP status codes>> for general
 If the process with the specified identifier does not exist on the server, see requirement /req/core/process-exception/no-such-process from <<OAProc-1>>.
 
 If the process exist but is immutable then <<req_deploy-replace-undeploy_deploy_response-immutable,/req/deploy-replace-undeploy/deploy/response-immutable>> applies.
+
+[[deploy-replace-undeploy-package]]
+=== Retrieving the formal description
+
+For every process deployed to the server, it is possible to retrieve its formal description. It corresponds to the request's body used to deploy or replace the process.
```

Comment:
Perhaps this working is more clear?  "For every mutable process, it is possible to retrieve the application package that was used to deploy the process."

Reply:
The "mutable" process part raises a good question. What about immutable ones? I have some processes that are immutable, but still represented using CWL. The echo process is a good example of this, as it comes "pre-deployed" with the service.

Reply:
@fmigneault clients know which processes are mutable or not since there is a member in the process description that indicates that.  So, clients should not request the "package" of an immutable process since there is no guarantee that the "package" exists.  The process may, for example, be compiled right into the executable of the server.  
In the event that a client does ask for the package for an immutable process then they should be prepared to get a 404.  I am not saying that they will get a 404; if the server has the package or can generate it (as is your case), then the server can offer it up but in the event that a package does not exist the client should be prepared to get a 404.  At least that is was I think should happen.  Thoughts?

Reply:
Yes. That is reasonable.